test/datafile.test.js

```diff
@@ -497,7 +498,8 @@ describe('Reader()', function(){
             var count = 0;
             var fileStream = fs.createReadStream(__dirname + "/data/log.deflate.avro");
 
-            fileStream.pipe(DataFile.Reader())
+            reader = fileStream.pipe(DataFile.Reader());
```

Comment:
This `reader` seems to be in the global scope - cause problems down the line for other reader tests